matter/si91x/siwx917/BRD4325x/autogen/sl_iostream_init_si91x_uart.c

```diff
@@ -0,0 +1,61 @@
+#if defined(SL_COMPONENT_CATALOG_PRESENT)

```

Comment:
I believe the sl_iostream is currently only used with RTT, not uart. We might use iostream uart for matter shell/ logs later so it might be ok to have It here...

@rcasallas-silabs what do you think

Reply:
Noticed I added uart/vcom inst, changed that to RTT via the sample app generation, after adding the component `iostream_rtt`

Reply:
I believe you're both correct. the RTT configuration looks ok now